packages/core/src/dialects/snowflake/query-generator-internal.ts

```diff
@@ -0,0 +1,28 @@
+import { AbstractQueryGeneratorInternal } from '../abstract/query-generator-internal.js';
+import type { AddLimitOffsetOptions } from '../abstract/query-generator.types.js';
+import type { SnowflakeDialect } from './index.js';
+
+const TECHNICAL_SCHEMA_NAMES = Object.freeze(['INFORMATION_SCHEMA', 'PERFORMANCE_SCHEMA', 'SYS', 'information_schema', 'performance_schema', 'sys']);
```

Comment:
I really wish that we could test on snowflake, since this very likely is incorrectly copied from MySQL/MariaDB. `INFORMATION_SCHEMA` seems to be correct, but the others don't.
Nothing we can safely do for this right now unfortunately.